Documentation/writing-custom-queries.md

```diff
@@ -212,19 +212,30 @@ metadata:
 spec:
   reportDataSources:
   - "unready-deployment-replicas"
+  view:
+    disabled: true
   columns:
+  - name: period_start
+    type: timestamp
+  - name: period_end
+    type: timestamp
   - name: namespace
     type: string
   - name: deployment
     type: string
-  - name: total_replica_unready_seconds,
+  - name: total_replica_unready_seconds
     type: double
-  - name: avg_replica_unready_seconds,
+  - name: avg_replica_unready_seconds
     type: double
+  inputs:
+  - name: ReportingStart
+  - name: ReportingEnd
   query: |
     SELECT
-        labels['namespace'] as namespace,
-        labels['deployment'] as deployment,
+        timestamp '{| default .Report.ReportingStart .Report.Inputs.ReportingStart | prestoTimestamp |}' AS period_start,
```

Comment:
I was thinking we would add these in the next example. This one is intended to work without input.

Reply:
Add adding an explanation to the section below:

> However, the above example is missing one crucial bit, and that's the ability to constrain the period of time that this query is reporting over.

Updating that to also mention that we need to expose the period being reported over (in addition to filtering)

Reply:
I moved the timestamp inputs out of the non-input example and added them to the final `ReportGenerationQuery` snippet later in the doc. 

I added a link to the [reports.md](https://github.com/operator-framework/operator-metering/blob/master/Documentation/report.md#reportingstart) that covers ReportingStart, ReportingEnd, and spec.inputs.